docs/my-website/docs/tutorials/vertex_ai_pay_go.md

```
@@ -0,0 +1,151 @@
+import Tabs from '@theme/Tabs';
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)